src/main/java/frc/robot/maps/FrostBiteMap.java

```diff
@@ -133,7 +136,16 @@ public IntakeData.Map getIntakeMap() {
         CSTalonSRX intakeMotor = new CSTalonSRX(9);
         intakeMotor.setInverted(true);
         RevDSolenoid intakeSolenoid = new RevDSolenoid(8, 9);
+        intakeMotor.getMotorController().configContinuousCurrentLimit(35);
+        intakeMotor.getMotorController().configPeakCurrentLimit(35);
+
+        final LinearFilter currentFilter = LinearFilter.singlePoleIIR(1, 0.01);
+        intakeMotor.addValidator(() -> {
+            final double current = Arrays.stream(intakeMotor.getCurrentAmps()).reduce(Double::sum).orElse(0.0);
```

Comment:
Do we actually want the sum, or do we want the average?

Reply:
An average is pretty much just a sum and we only have one motor. If we happen to add any more we could just multiply the current limit.